src/App.js

```diff
@@ -1,25 +1,16 @@
-import logo from './logo.svg';
+import { BiCurrentLocation } from 'react-icons/bi';
+
 import './App.css';
 
-function App() {
+export default function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="wrapper">
+      <div className=""></div>
+      <button className="nav-location-btn">
+        <span className="nav-location-icon">
+          <BiCurrentLocation size={27} />
+        </span>
+      </button>
     </div>
   );
 }
-
-export default App;
```

Comment:
the separate `export default` will help you easier to add `defaultProps` and `propsType`